packages/bsky/src/views/index.ts

```diff
@@ -1128,6 +1139,16 @@ export class Views {
     return true
   }
 
+  viewerPinned(uri: string, state: HydrationState) {
+    const authorDid = creatorFromUri(uri)
```

Comment:
This function fully parses the at-uri under the hood before returning the authorDid. It's a bit unfortunate to have to do a second parse for every single post uri that we hydrate from now on. 

it's probably not a huge deal, but I do wonder if there's a way around it.

Ideas that come to mind are: 
- doing a simple string comparison instead (`uri.includes(state.ctx.viewer)`) for instance
- piping through the post creator from the `post()` view function where it's already been parsed out so we don't have to do it again down here

Reply:
Yeah this makes sense, I think passing it in from `post()` makes the most sense